includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -542,18 +583,20 @@ public static function add_post_columns( $columns ) {
 		$columns = array_merge(
 			$columns,
 			array(
-				'error_status' => esc_html__( 'Error Status', 'amp' ),
-				AMP_Validation_Error_Taxonomy::REMOVED_ELEMENTS => esc_html__( 'Removed Elements', 'amp' ),
-				AMP_Validation_Error_Taxonomy::REMOVED_ATTRIBUTES => esc_html__( 'Removed Attributes', 'amp' ),
-				AMP_Validation_Error_Taxonomy::SOURCES_INVALID_OUTPUT => esc_html__( 'Incompatible Sources', 'amp' ),
+				'error_status' => sprintf( '%s<span class="dashicons dashicons-editor-help"></span>', esc_html__( 'Status', 'amp' ) ),  // @todo Create actual tooltip.
```

Comment:
The `'error_status'` here can be replaced with the `AMP_Validation_Error_Taxonomy::ERROR_STATUS` constant I believe.